Qumran.sln.DotSettings.user

```diff
@@ -11,4 +11,7 @@
 	<s:String x:Key="/Default/Environment/AssemblyExplorer/XmlDocument/@EntryValue">&lt;AssemblyExplorer&gt;
   &lt;Assembly Path="/Users/bronson/.nuget/packages/nettopologysuite/2.0.0/lib/netstandard2.0/NetTopologySuite.dll" /&gt;
 &lt;/AssemblyExplorer&gt;</s:String>
+	<s:String x:Key="/Default/Environment/UnitTesting/UnitTestSessionStore/Sessions/=91b9dfa2_002D0c15_002D4a3d_002Da870_002D361628bfb78c/@EntryIndexedValue">&lt;SessionState ContinuousTestingIsOn="False" ContinuousTestingMode="0" FrameworkVersion="{x:Null}" IsLocked="False" Name="All tests from &amp;lt;sqe-api-test&amp;gt;" PlatformMonoPreference="{x:Null}" PlatformType="{x:Null}" xmlns="urn:schemas-jetbrains-com:jetbrains-ut-session" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"&gt;
```

Comment:
@zmbq, do you know what this is? It must be added by my IDE, but it looks unnecessary. I will try deleting it if you think that is ok.

Reply:
Hmmm. *.user is already in the .gitignore file, I wonder how this got in then.

Reply:
OK, I just deleted the file and there seems to be no problem.